base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -249,18 +249,31 @@ impl ConsensusConstants {
 
     pub fn stibbons() -> Vec<Self> {
         let mut algos = HashMap::new();
-        // seting sha3/monero to 40/60 split
+        // Previously these were incorrectly set to `target_time` of 20 and 30, so
+        // most blocks before 1400 hit the minimum difficulty of 60M and 60k
+        // algos.insert(PowAlgorithm::Sha3, PowAlgorithmConstants {
+        //     max_target_time: 1800,
+        //     min_difficulty: 60_000_000.into(),
+        //     max_difficulty: u64::MAX.into(),
+        //     target_time: 30,
+        // });
+        // algos.insert(PowAlgorithm::Monero, PowAlgorithmConstants {
+        //     max_target_time: 1200,
+        //     min_difficulty: 60_000.into(),
+        //     max_difficulty: u64::MAX.into(),
+        //     target_time: 20,
+        // });
         algos.insert(PowAlgorithm::Sha3, PowAlgorithmConstants {
             max_target_time: 1800,
             min_difficulty: 60_000_000.into(),
-            max_difficulty: u64::MAX.into(),
-            target_time: 30,
+            max_difficulty: 60_000_000.into(),
```

Comment:
This does not seem to be right, why is the `max_difficulty = min_difficulty` now?

Reply:
Comment in line 252 explains this, perhaps I can make it more clear?

Reply:
Basically because the target time was so low, there are blocks where the difficulty was the `min_difficulty`

Reply:
Looking at it again, I would say the comment is not conveying the reason that this condition, `max_difficulty = min_difficulty`, should be implemented, being that we want to ensure old blocks mined up to 1400 are still accepted when validated.